e2e-test/tsps_e2e_test.py

```diff
@@ -0,0 +1,395 @@
+from workspace_helper import create_workspace, delete_workspace
+from app_helper import create_app, poll_for_app_url
+import argparse
+import requests
+import os
+import json
+import uuid
+import time
+import logging
+
+
+# Upload data to WDS using APIs
+def upload_wds_data_using_api(wds_url, workspace_id, tsv_file_name, record_name):
+    #open TSV file in read mode
+    with open(tsv_file_name) as tsv_file:
+        request_file = tsv_file.read()
+
+    uri = f"{wds_url}/{workspace_id}/tsv/v0.2/{record_name}"
+    headers = {"Authorization": f"Bearer {azure_user_token}"}
```

Comment:
`azure_user_token` is not currently an input to this function, we should make it one rather than effectively using a global variable